dlt/dlt_broker.py

```diff
@@ -187,9 +220,13 @@ def remove_context(self, context_queue):
 
     def stop(self):
         """Stop the broker"""
-        logger.info("Stopping DLTContextHandler and DLTMessageHandler")
+        logger.info("Stopping DLTContextHandler and %s", type(self.msg_handler).__name__)
+
+        if isinstance(self.msg_handler, DLTFileSpinner):
+            logger.debug("Stop iterating to file %s", self._filename)
+            self.msg_handler.stop()
```

Comment:
It's weird. Why DLTMessageHandler does not need to stop?

if it does not need, why not create an empty function in DLTMessageHandler then avoid the if block?